code/element-id/octane.js

```diff
@@ -0,0 +1,6 @@
+import Component from '@glimmer/component';
+import { guidFor } from '@ember/object/internals';
+
+export default class InputTextComponent extends Component {
+  inputId = 'textInput-' + guidFor(this); 
```

Comment:
```js
get inputId() {
  return `'textInput-${guidFor(this)}`;
}
```

Reply:
Since this is a static field that won’t change, using a class field is probably better here